kotlin-native/Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -105,9 +105,9 @@ private fun List<String>?.isTrue(): Boolean {
     return this?.last() == "true"
 }
 
-private fun runCmd(command: Array<String>, verbose: Boolean = false) {
+private fun runCmd(command: Array<String>, verbose: Boolean = false, redirectInputFile: File? = null) {
```

Comment:
Please add `redirectInputFile` into the log printed in the verbose mode above.
This way, if the command fails, users will at least be able to reproduce this.

Reply:
Done.